torch/__init__.py

```diff
@@ -1586,6 +1589,37 @@ def __call__(self, model_, inputs_):
         return self.compiler_fn(model_, inputs_, **self.kwargs)
 
 
+################################################################################
+# Exposing torch.export() and related support APIs
+################################################################################
+from ._constraint import Constraint
+
+def export(
+    f: Callable,
+    args: Tuple[Any],
+    kwargs: Optional[Dict[str, Any]] = None,
+    constraints: Optional[List[Constraint]] = None,
+) -> "torch.export_utils.ExportedProgram":
+    """
+    Traces either an nn.Module's forward function or just a callable with PyTorch
+    operations inside and produce a ExportedProgram.
+
+    Args:
+        m: the `nn.Module` or callable to trace.
```

Comment:
If I export an nn.Module, what happens to the parameters? This also needs to be spelled out here.